sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/__init__.py

```diff
@@ -1,26 +0,0 @@
-# --------------------------------------------------------------------------
```

Comment:
out of curiosity, the approach used here (removing __ init __.py) is called [native namespace package](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/#native-namespace-packages)?

Reply:
I see, since we're now python 3 only, we are free to use the native nspkg pattern. nice!